### ISSUE TITLE
Getting lots of "deprecated: call #name instead" messages

### ISSUE THREAD

**Issue by charlesdeb** [cucumber/cucumber-ruby#942]
I have looked at the source of `running_test_case.rb` and can see what is causing all these deprecation warnings (three per scenario). Apparently somewhere in my code/config (or in the code of a gem that I am using) something is calling this 'title' method instead of a 'name' method. However, I have no idea quite what piece of code is calling this; it's beyond my current ruby debugging skills! My tests all seem to run fine - but getting three lines of deprecation warning for every scenario creates a **lot** of noise. The issue seems a little like #232.

**Comment by mattwynne** [comment]
Sorry about the noise, can you post your Gemfile and maybe we can see what's likely to be causing it?

Does the deprecation warning include a stack trace? If not we can add one.

**Comment by charlesdeb** [comment]
Thanks for getting back to me. Here goes the gemfile - I am using RefineryCMS and am trying to upgrade to version 3. Had no issues before the upgrade - but the upgrade has affected quite a few gems. I am getting no stack trace for deprecation warnings. Please let me know how I can switch it on for cucumber.

```
source 'https://rubygems.org'
ruby "2.1.3"

gem 'rails'
gem 'pg'
gem 'fog'
gem 'delayed_job_active_record' #, "=0.3.3"  # without PG 9.1 you get lock errors
gem "workless", "~> 1.1.3"
gem "daemons"
gem 'jquery-rails'
gem 'carmen-rails', '~> 1.0.0'
gem 'gibbon' #, '~> 0.3.5'
gem 'memcachier'
gem 'dalli'
gem 'twitter'
gem 'sass', '~> 3.2.15'  # as per https://github.com/sass/sass/issues/1162, errors while saving cache at Heroku, can't dump anonymous class
gem "acts-as-taggable-on"  # needed for rake tasks for refinerycms-blog

group :development, :test do
  gem 'yaml_db', :git => "git://github.com/lostapathy/yaml_db.git"
  gem 'refinerycms-testing' #, '~> 2.1.2'
end

group :test do
  gem 'cucumber' 
  gem 'cucumber-rails', require: false # required code for refinerycms 2.1.2 upgrade
  gem 'capybara-webkit'
  gem 'launchy'
  gem 'turn', :require => false
  gem 'minitest'
  gem 'email_spec'
  gem 'rspec-activemodel-mocks'
end

group :production, :staging do
  gem 'newrelic_rpm'
  gem 'rails_12factor'
  gem 'heroku-deflater', '0.5.2'  # enables gzip compression for rails on heroku, locked to 0.5.2 because of https://github.com/romanbsd/heroku-deflater/issues/15
  gem "dragonfly-s3_data_store"
end

gem 'unicorn'

# Refinery CMS
gem 'refinerycms', '~>3.0.0'

gem 'refinerycms-blog', :git => 'git://github.com/refinery/refinerycms-blog.git', :branch => 'master'
gem 'refinerycms-inquiries', :git => 'git://github.com/refinery/refinerycms-inquiries.git', :branch => 'master'
gem 'refinerycms-authentication-devise', '~> 1.0'

gem 'refinerycms-prayer_points', :path => 'vendor/extensions'
gem 'refinerycms-tours', :path => 'vendor/extensions'
```

**Comment by mattwynne** [comment]
I've just released v2.3.1 which should give you a better deprecation warning with the file and line number of the code that's calling `#title`. Can you upgrade and tell us who the culprit is?

**Comment by charlesdeb** [comment]
Many thanks - that worked a treat. It /was/ in my own code, and now that I see where the problem was, I shouldn't have assumed a blanket search over my own project for the word "title" wouldn't have worked... But this change you've implemented should help other folks experiencing deprecation warnings in the future.
